src/ProposalTypesConfigurator.sol

```diff
@@ -20,15 +20,21 @@ contract ProposalTypesConfigurator is IProposalTypesConfigurator {
     //////////////////////////////////////////////////////////////*/
 
     mapping(uint8 proposalTypeId => ProposalType) internal _proposalTypes;
+    mapping(uint8 proposalTypeId => bool) internal _proposalTypesExists;
+    mapping(uint8 proposalTypeId => Scope[]) public scopes;
+    mapping(uint8 proposalTypeId => mapping(bytes32 typeHash => bool)) public scopeExists;
 
     /*//////////////////////////////////////////////////////////////
                                MODIFIERS
     //////////////////////////////////////////////////////////////*/
 
     modifier onlyAdminOrTimelock() {
-        if (msg.sender != governor.admin() && msg.sender != governor.timelock()) {
-            revert NotAdminOrTimelock();
-        }
+        if (msg.sender != governor.admin() && msg.sender != governor.timelock()) revert NotAdminOrTimelock();
+        _;
+    }
+
+    modifier onlyAdmin() {
+        if (msg.sender != governor.admin()) revert NotAdmin();
         _;
     }
```

Comment:
confirm existence and usage of `onlyAdmin`

Reply:
Just got confirmation from @yitongzhang that this should just be the admin role for now